src/foundry/client-esm/applications/api/application.d.mts

```diff
@@ -1,10 +1,38 @@
+import type { MustConform } from "../../../../types/helperTypes.d.mts";
 import type { DeepPartial, InexactPartial, MaybePromise } from "../../../../types/utils.d.mts";
 import type EventEmitterMixin from "../../../common/utils/event-emitter.d.mts";
 
 // TODO: Investigate use of DeepPartial vs Partial vs InexactPartial
 // TODO: Should
```

Comment:
This is from me, I think this is supposed to be "should renderContext be added to the generics", which you've done here so you can remove this line.